charlesinto/iReporter
# #162533048 Feat update redflag location

#### What does this PR do?

- Update redflag location

#### Description of Task to be completed?
- implement jwt authentication
- verify user token
- user should be able to post new report

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `PATCH` request to `http://localhost:7000/api/v1/red-flags/:id/location`
- You should receive a successful response

#### Any background context you want to provide?
- the PATCH endpoint response should be structured like this- 

`
{
    "location":"5.24434,6.7786"
}
`

`

#### What are the relevant pivotal tracker stories?
[#162533048](https://www.pivotaltracker.com/story/show/162533048)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/26615085/49800375-9fc82700-fd47-11e8-8f06-4d3d76d040cd.png)




#### Questions:
N/A
